camp/sour1emon/XBox Controller Left Stick Y direction/src/main/java/frc/robot/commands/MotorDefaultCommand.java

```diff
@@ -0,0 +1,35 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj.XboxController;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.MotorSubsystem;
+
+/**
+ * A command that sets the speed of a motor subsystem based on the Y direction
+ * of the left stick of an Xbox controller.
+ */
+public class MotorDefaultCommand extends Command {
+    private XboxController controller;
```

Comment:
We don't want commands to take in a whole XboxController because it's not very reusable. What if we want to change to a different controller, button board, or sensor-based control later on? 

We'd like this command to take in only what it needs. In this case, it only needs to take in a doubleSupplier. 

Here's an example from Ted that uses the correct pattern:
[ted/src/main/java/frc/robot/commands/ShooterSpinUpCommand.java](https://github.com/Team4682CyBears/Crescendo2024/blob/70c472825276ae1bd7ec6ff5325898e8c4da4e88/ted/src/main/java/frc/robot/commands/ShooterSpinUpCommand.java#L1)